port/linux/storage.c

```diff
@@ -80,8 +81,10 @@ oc_storage_write(const char *store, uint8_t *buf, size_t size)
   if (!fp)
     return -EINVAL;
 
-  size = fwrite(buf, 1, size, fp);
+  size_t wsize = fwrite(buf, 1, size, fp);
```

Comment:
@jkralik do we want to handle all possible errors here? 

* size != wsize (when `fwrite` fails)
* `fflush`, `fsync`, `fileno`, `fclose` failures

I guess we should.

Reply:
yes we want

Reply:
yes, there's not much to do when `fclose`/`close` fail, I guess we can log it.